messaging.rst

```diff
@@ -14,22 +14,16 @@ Data Structures
 Offchain Balance Proof
 ----------------------
 
-Data required by the smart contracts to update the payment channel end of the participant that signed the balance proof.
-Messages into smart contracts contain a shorter form called :ref:`Onchain Balance Proof <balance-proof-onchain>`.
+An offchain balance proof is a set of fields of Raiden protocol messages.  The fields contain enough information for producing an :ref:`Onchain Balance Proof <balance-proof-onchain>`, which is required by the smart contracts to update the payment channel (especially, the state of the participant that signed the balance proof).  The offchain balance contains more information for easier bookkeeping.
```

Comment:
nitpick: 

> which is required by the smart contracts to update the payment channel (especially, the state of the participant that signed the balance proof)

As far as the contracts are concerned the state of the participant that signed the balance proof is the only thing that can be updated from this.

Reply:
You have double space in two places: `messages.  The fields`, `proof).  The offchain`

Reply:
Will fix in this PR.